src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,191 @@
+import { Component } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Table, {
+  TableBody,
+  TableHead,
+  TableRow,
+  TableSortLabel,
+  TableCell,
+} from 'material-ui/Table';
+
+const columns = ['Project', 'Summary', 'Tag', 'Assignee', 'Last Updated'];
+const styles = theme => ({
+  table: {
+    marginTop: theme.spacing.unit * 3,
+    width: '100%',
+    overflowX: 'auto',
+  },
+  summary: {
+    whiteSpace: 'nowrap',
+  },
+});
+
+class BugsTable extends Component {
+  state = {
+    order: 'desc',
+    orderBy: 'Last Updated',
+    data: [
+      {
+        project: 'Servo',
```

Comment:
If `BugsTable` is a component, then it should be general enough to be reused with different data. Put the data in `views/Project/index.jsx` and send it via component props.

```jsx
<BugsTable items={...} columns={...} ... />
```

Feel free to inspire yourself by looking at `tc-web`.